ui/src/components/SubDescription/SubDescription.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+import SubDescription from './SubDescription';
+
+it('Sub Description component - without links', async () => {
+    const props = {
+        text: `Some description without any break or changes`,
+    };
+    render(<SubDescription {...props} />);
+
+    const subDesc = await screen.findByText(props.text);
+    expect(subDesc).toBeInTheDocument();
+});
+
+it('Sub Description component - links', async () => {
+    const props = {
+        text: `Ingesting data from to Splunk Cloud?</br>Read our [[blogPost]] to learn more about Data Manager and it's availability on your Splunk Cloud instance.`,
+        links: [
+            {
+                slug: 'blogPost',
+                link: 'https://splk.it/31oy2b2',
+                linkText: 'blog post',
+            },
+        ],
+    };
+    render(<SubDescription {...props} />);
+
+    const firstParagraph = await screen.findByText('Ingesting data from to Splunk Cloud?');
+    expect(firstParagraph).toBeInTheDocument();
+    const wholeSubDescription = firstParagraph.parentNode;
+    expect(wholeSubDescription?.textContent).toEqual(
+        props.text.replaceAll('</br>', '').replaceAll('[[blogPost]]', 'blog post')
```

Comment:
is a requirement or mistypo?
markdown supports HTML tag _br_, which can be represented like `<br>` or `<br />`.
https://spec.commonmark.org/0.29/#hard-line-breaks
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

Reply:
tldr: that was on purpose as i use that to split text in component, but maybe should use <br> as more ppl are familiar with it.

yea that one was kind of on purpose, but now when i think about it maybe it actually does not make sense, as yea HTML supports that one but i did not want to use there HTML injection as i wanted to have a full control of what is going on with that description and that ppl do not use some weird html code there (probably not something popular but i would go with if thats not necessary lets do not do it). So thats why i created something similar (</br>) to not give the idea that html is supported
And it does not make sense as probably more ppl would be keen to use those two (<br> or <br />)

also you can check that in component itself i just split text by that </br> 
```
            return text.split('</br>').map((elem, ind) => (
                // using index as key for elements as they dont have unique values but are unique itself
                <span key={`subDesc${i}${ind}`}>
                    {ind > 0 && ind < text.length - 1 && <br />}
                    {elem}
                </span>
            ));
```
so i do not sue HTML injection but more like create own patter to use that description.

Reply:
wdyt about it, maybe actually a better idea will be to use something like <breakline>, just to be more clear, that it is not html but also have a clear syntax

Reply:
why not standard `\n`? It should work out of the box without a need to replace that if we adjust CSS `white-space`

Reply:
i have absolutely not idea, but my brain just forgot about that way..., yea thats way better

Reply:
<br> replaced with \n